### ISSUE TITLE
WebUI constantly and randomly refreshes

### ISSUE THREAD

**Issue by erichter2018** [HaveAGitGat/Tdarr#407]
**Describe the bug**
WebUI constantly and randomly refreshes.

**To Reproduce**
open WebUI, wait a little bit

**Expected behavior**
Not randomly refresh

 - OS: Windows 10
 - Browser Chrome, Safari
 - Version 2.00.07 - new issue, wasn't there on prior ones

**Comment by HaveAGitGat** [comment]
What do you mean by refreshes? As in the page fully reloads or just the connection splash screen shows? I think it's due to the server being slow so some of the requests timeout (can happen if Tdarr is run on an HDD or if there is lots of scanning/transcoding happening). Should be improved in 2.00.08 which is still in pre-release but can be downloaded by setting `2.00.08` in the config version.

```
{
  "platform_arch": "default",
  "modules": {
    "Tdarr_Desktop": {
      "enabled": true,
      "requiredVersion": "2.00.08",
      "currentVersion": ""
    },
    "Tdarr_Node": {
      "enabled": true,
      "requiredVersion": "2.00.08",
      "currentVersion": "2.00.08"
    },
    "Tdarr_Server": {
      "enabled": true,
      "requiredVersion": "2.00.08",
      "currentVersion": "2.00.08"
    },
    "Tdarr_WebUI": {
      "enabled": true,
      "requiredVersion": "2.00.08",
      "currentVersion": ""
    }
  }
}
```

**Comment by HaveAGitGat** [comment]
I've heard this is fixed in 2.00.08 so will close this.
